README.md

```diff
@@ -0,0 +1,62 @@
+Tours & Travel Agency
+=====================
+
+The project represents a website, which offers customers different tours all around the world
+
+###User performs following functions:
+
+   * registration
+   * logging in
+   * choosing an offer and making an order
+    
+
+###Admin performs following functions:
+   * logging in
+   * adding new offers to catalogues
+   * deleting offers
+
+
+###Getting Started
+ 
+1 Clone or download this repository
+
+> git clone https://github.com/Cisfinitum/FirstProject.git
+>
+2 Build project
+  > gradle build
+  >
+Or if you did not download Gradle you can use wrapper
+> gradle wrapper --gradle-version 5.1 
+>
+> gradle build
+>
+3 Start Server
+ 
+copy .war file in directory %CATALINA_HOME%\webapps (example: C:\tomcat\webapps)
+>%CATALINA_HOME%\bin\startup.bat
+>
+See it up and running on http://localhost:8080
+
+
+#### Also you can use IntelliJ IDEA
+
+* File -> new -> Project from version control -> https://github.com/Cisfinitum/FirstProject.git
+* Build project
+* * Edit configuration -> Application server -> path to Tomcat on your computer 
+
+  * also you can use Tomcat plugin
+  
+Add to build.gradle
+
+>dependencies {
+    def tomcatVersion = '9.0.1'
+    tomcat "org.apache.tomcat.embed:tomcat-embed-core:${tomcatVersion}",
+           "org.apache.tomcat.embed:tomcat-embed-logging-juli:9.0.0.M6",
+           "org.apache.tomcat.embed:tomcat-embed-jasper:${tomcatVersion}"
+}
+>
+ >apply plugin: 'com.bmuschko.tomcat'
+>
+* Run 
+
```

Comment:
Can you please add here an example with up and run the app using local tomcat and intellij configuration for it